### DISCUSSION TITLE
Add more info to the returned list of issues

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/hacktoberfest-issue-hunt#4] [Ideas]
Right now, the `npm` module returns only `title` and `link`. Shall we add other fields like `labels`, `language`, etc?

> Although you can pass an additional option `complete: true` to get the entire issue object with the current implementation. 

@heysujal, @morelmiles, @lbaweja1999, @ishikkkkaaaa, @Gauravmauryaios, @StarTrooper08, @san-coding your thoughts on this? 

Let me know what kind of information would you like to consume? **I will make another release to support those.**

**Comment by lbaweja1999** [comment]
Yes it would be great

**Comment by atharvashirdhankar** [comment]
> Right now, the `npm` module returns only `title` and `link`. Shall we add other fields like `labels`, `language`, etc?
> 
> > Although you can pass an additional option `complete: true` to get the entire issue object with the current implementation. 
> 
> @heysujal, @morelmiles, @lbaweja1999, @ishikkkkaaaa, @Gauravmauryaios, @StarTrooper08, @san-coding your thoughts on this? 
> 
> Let me know what kind of information would you like to consume? **I will make another release to support those.**

Yes both the fields (labels and language)will be useful 🚀.

**Comment by vinitshahdeo** [comment]
Thanks for the feedback folks. I have made the release. Please refer [`v2.0.0` release](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt/releases/tag/v2.0.0).
